### ISSUE TITLE
Setting node memory size with docker-compose yaml configuration

### ISSUE THREAD

**Issue by jason19970210** [emqx/emqx#7224]
Once I up the docker-compose with my yaml, then access to web console, the 'Memory Info' is always set to 150-180 M total space with 90~110 M memory usage.

Upon the usage will always over the 70% high watermark alert.

Here is my system config:
OS :                                    Ubuntu 20.04.4 LTS
RAM :                                 32GB ( via `free -h` )
Docker Ver. :                     Docker version 20.10.12, build e91ed57
Docker-Compose Ver. :  Docker Compose version v2.2.1

also with my docker-compose.yml
```
# docker-compose-emqx
version: "3"

volumes:
  vol-emqx-data:
    name: emqx-data
  vol-emqx-etc:
    name: emqx-etc
  vol-emqx-log:
    name: emqx-log

services:
  emqx1:
    image: emqx/emqx:4.4.1
    environment:
      - "EMQX_NAME=emqx_1"
      - "EMQX_HOST=node1.emqx.io"
      - "EMQX_CLUSTER__DISCOVERY=static"
      - "EMQX_CLUSTER__STATIC__SEEDS=emqx@node1.emqx.io"
      - "EMQX_ZONE__EXTERNAL__RETRY_INTERVAL=2s"
      - "EMQX_MQTT__MAX_TOPIC_ALIAS=10"

    volumes:
      - vol-emqx-data:/opt/emqx/data
      - vol-emqx-etc:/opt/emqx/etc
      - vol-emqx-log:/opt/emqx/log

    networks:
      emqx-bridge:
        aliases:
          - node1.emqx.io

    ports:
      - 1883:1883
      - 8081:8081
      - 8083:8083
      - 8883:8883
      - 18083:18083

networks:
  emqx-bridge:
    driver: bridge

```

Running Info:
| Name | Erlang OTP Release | Erlang Processes | CPU Info | Memory Info | MaxFds | Status |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- |
| emqx_1@node1.emqx.io | 24.1.5/12.1.5 | 452 / 2097152 | 0.03 / 0.04 / 0.04  | 108.43M / 175.75M  | 1048576 |  Running |

**Comment by thalesmg** [comment]
Hi @jason19970210 !

The "total" value in "Memory Info" is a bit misleading: it means the total amount of memory currently allocated by the Erlang VM, which can be higher than the actually used memory by processes in the VM.

**Comment by jason19970210** [comment]
@thalesmg 
By the meaning for your apply, so this memory allocated is dynamic showing on the console panel ?
thanks for your apply !

**Comment by thalesmg** [comment]
Yes, the panel data comes from periodically collecting various metrics from the VM, such as memory, which will change over time.

**Comment by qzhuyan** [comment]
@jason19970210 we have a bug about false alarm for "70%` high watermark" the fix has been applied and will be released in 4.4.2

**Comment by qzhuyan** [comment]
@matthewGEA 
4.4.2 fixed the false alarm during startup.

However there is another issue in OTP that take cached memory into account: see https://github.com/erlang/otp/pull/5800

you could check with `free -m` in Linux to find the memory usage in cache.

Also we have a fix in version 4.4.4 that eliminates the issue above.
